### ISSUE TITLE
Voting on issues

### ISSUE THREAD

**Issue by aramisf** [FForstbach/great-city#1]
Start recording votes that users give for issues.
Each user must be able to vote only once.

Question open to discussion:

1. Will users be able to unvote on a specific issue?

**Comment by philippleo** [comment]
I think unvote should be implemented. Sometimes people click by mistake. Unvote should be disabled if we reach a sponsor goal (200 votes)

**Comment by aramisf** [comment]
How about not locking the voting counter up, after reaching the goal? Instead we could change the issue's state.

Voting then would only rate issue's popularity.

Tell me your thoughts :)

**Comment by philippleo** [comment]
Definitely what I, to clarify "unvote" is disabled and "vote" stays enabled

**Comment by FForstbach** [comment]
I think instead of voting could be something like "LIKE", they way benni designed it in the mockups. so user is able to toggle the LIKE button

**Comment by philippleo** [comment]
We can call it "LIKE" but for me it feels and sounds more honest and binding to vote for something instead of like.

**Comment by aramisf** [comment]
Yeah, I would also choose the term `vote` to describe the action. The main reason is that we need to know how many people are endorsing an action to be taken.

Also, it is good to remember that we will need some authentication mean to count the votes.
